ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -139,6 +139,20 @@ tests:
     test:
     - chain: openshift-upgrade-qe-sanity
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy
+- as: gcp-ipi-proxy-etcd-encryption-f28
+  cron: 11 10 28 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:arm64-target
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      ENABLE_OTA_TEST: OCP-69948
```

Comment:
I guess we do not need this line.

Reply:
what's the case for? I see the test case are enabled in amd64 files. https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml#L91

Reply:
This is for specific OTA test case coverage, for the newly added jobs, does not need it. If needed, OTA team should be responsible for updating the job config.

Reply:
Got it, thanks.